packages/imageviewer/style/index.css

```diff
@@ -10,13 +10,18 @@
 
 
 .jp-ImageViewer > div {
-  transform-origin: top left;
+  position: absolute;
+  transform-origin: center center;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
 }
 
 
 .jp-ImageViewer img {
-  max-width: 100%;
-  max-height: 100%;
+  display: table-cell;
```

Comment:
Are these CSS properties (`display`, `vertical-align`, `text-align`) necessary here? I don't quite understand what they are meant to do.

Reply:
They were left over from getting the image to center properly. The div attributes to that now, so these aren't necessary.